src/pybind/mgr/dashboard/frontend/src/app/shared/enum/grafana-template.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum GrafanaStyles {
```

Comment:
This is only being used in 1 place.
I would recommend either to remove this enum or use it everywhere.

Reply:
ack